.circleci/config.yml

```diff
@@ -1179,6 +1179,9 @@ jobs:
       - checkout
       - react-get-deps
       - run: make -C webui/react test-ci
+      - run: curl -Os https://uploader.codecov.io/latest/linux/codecov
+      - run: chmod +x codecov
+      - run: ./codecov -t fd979617-a8d7-4fc5-900e-6999db718fea
```

Comment:
oh nice, I had no idea that they [deprecated their bash uploader](https://docs.codecov.com/docs/about-the-codecov-bash-uploader), good to know!